recipes/unicorn-binance-rest-api/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "unicorn-binance-rest-api" %}
+{% set version = "1.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unicorn-binance-rest-api-{{ version }}.tar.gz
+  sha256: 51aa63db35cfd55d7766c7f2a54bc4dd7653dbfebde03d2ab6fa1c285daa8321
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - certifi
+    - colorama
+    - cryptography
+    - dateparser
+    - pyopenssl
+    - python >=3.6
+    - regex <=2022.3.2
+    - requests
+    - service_identity
+    - ujson
+  run:
+    - certifi
+    - colorama
+    - cryptography
+    - dateparser
+    - pyopenssl
+    - python >=3.6
+    - regex <=2022.3.2
+    - requests
+    - service_identity
+    - ujson
+
+test:
+  imports:
+    - unicorn_binance_rest_api
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-rest-api
+  summary: An unofficial Python API to use the Binance REST API`s (com+testnet, com-margin+testnet, com-isolated_margin+testnet, com-futures+testnet, us, tr) in a easy, fast, flexible, robust and fully-featured way.
+  license: MIT
+  license_file: LICENSE
+  dev_url: https://www.lucit.tech
+  doc_url: https://unicorn-binance-rest-api.docs.lucit.tech/
+  doc_source_url: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-rest-api/blob/master/README.md
```

Comment:
This is available in conda-build but generally not used in conda-forge. To be honest, I don't even know if it shows up anywhere. I would take it out again, just to remove a possible point of failure.

Reply:
dev_url and doc_url works and is shown up here: https://anaconda.org/conda-forge/unicorn-fy

doc_source_url seems not be working

Reply:
Yes that would make sense. It is valid and can be used in a meta.yaml but not shown (anymore?).

Reply:
in your team chat someone told me its not supported by anaconda api atm... maybe in the future.